sfaira/estimators/keras.py

```diff
@@ -744,7 +744,7 @@ def generator():
                     for i in idx:
                         x_sample = self.data.X[i, :].toarray().flatten() if sparse else self.data.X[i, :].flatten()
                         sf_sample = prepare_sf(x=x_sample)[0]
-                        y_sample = self.data.obs[self._adata_ids.cellontology_id][i]
+                        y_sample = self.data.obs[self._adata_ids.cellontology_class][i]
```

Comment:
@davidsebfischer I remember you changing this line to cellontology_id recently. I can't quite determine what's correct here.

Reply:
`cellontology_id` failed for me, not sure if its because I was using an old `obs` file. Also, `cellontology_class` is being used in other conditions within this function so I assumed there was a mistake in this line.